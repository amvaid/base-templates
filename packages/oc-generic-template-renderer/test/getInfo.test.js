const getInfo = require('../lib/getInfo');
const packageJson = {
  name: 'oc-template-generic',
  version: '1.0.0',
  externals: {
    'lib-one': {
      global: 'LibOne',
      url: 'https://cdn.com/lib-one/1.0.0/lib-one.min.js'
    },
    'lib-two': {
      global: 'LibTwo',
      url: 'https://cdn.com/lib-two/1.0.0/lib-two.min.js'
    }
  }
};

test('should return the correct info', () => {
  const info = getInfo(packageJson);
  expect(info).toMatchSnapshot();
});
