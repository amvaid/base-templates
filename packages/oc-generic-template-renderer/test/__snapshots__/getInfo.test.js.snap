// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the correct info 1`] = `
Object {
  "externals": Array [
    Object {
      "global": "LibOne",
      "name": "lib-one",
      "url": "https://cdn.com/lib-one/1.0.0/lib-one.min.js",
    },
    Object {
      "global": "LibTwo",
      "name": "lib-two",
      "url": "https://cdn.com/lib-two/1.0.0/lib-two.min.js",
    },
  ],
  "type": "oc-template-generic",
  "version": "1.0.0",
}
`;
